export/export.py

```diff
@@ -0,0 +1,866 @@
+                "dynamic_shapes": self._dynamic_shapes,
+            }
+            return models, export_config
+
+        elif stage_name == "edge_transform_and_lower":
+            # For EdgeTransformAndLowerStage.run(exported_programs, transform_config)
+            exported_programs = self._exported_program
+            transform_config = {
+                "constant_methods": self._constant_methods,
+            }
+            return exported_programs, transform_config
+
+        elif stage_name == "executorch":
+            # For ExecutorchStage.run(edge_program, backend_options)
+            edge_program = self._edge_program_manager
+            backend_options = {}
+            return edge_program, backend_options
+
+        else:
+            return None, {}
+
+    def _store_stage_result(self, stage_name: str, result: Any) -> None:
+        """
+        Store the result of a stage.
+
+        Args:
+            stage_name: Name of the stage
+            result: Result of the stage
+        """
+        if stage_name == "source_transform":
+            self._model = result
+        elif stage_name == "quantize":
+            self._model = result
+        elif stage_name == "export":
+            self._exported_program = result
+        elif stage_name == "edge_transform_and_lower":
+            self._edge_program_manager = result
+            self._delegation_info = self._stages[stage_name].delegation_info
+        elif stage_name == "executorch":
+            self._executorch_program_manager = result
+
+    def _run_pipeline(self, start_stage: str) -> None:
+        """
+        Run the pipeline starting from a specific stage.
+
+        This method cascades through the pipeline of stages, starting from the specified stage.
+        Each stage's artifact is passed as input to the next stage.
+
+        Args:
+            start_stage: Name of the stage to start from
+
+        Raises:
+            ValueError: If the stage is not found
+        """
+        if start_stage not in self._stages:
+            raise ValueError(f"Stage '{start_stage}' not found")
+
+        # Find the starting index in the pipeline
+        try:
+            start_index = self._pipeline.index(start_stage)
+        except ValueError:
+            raise ValueError(f"Stage '{start_stage}' not found in pipeline")
+
+        # Set the current stage index
+        self._current_stage_index = start_index
+
+        # Process each stage in the pipeline
+        for i in range(start_index, len(self._pipeline)):
+            stage_name = self._pipeline[i]
+            self._current_stage_index = i
+
+            # Get the primary input and configuration parameters for this stage
+            primary_input, config_params = self._get_stage_inputs(stage_name)
+
+            # Run the stage
+            result = self._run_stage(stage_name, primary_input, config_params)
+
+            # Store the result
+            self._store_stage_result(stage_name, result)
```

Comment:
Yeah at thsi point i expect jsut iterate through stages and run without needing to figure out input and storage stage outputs. Outputs of each stage should be there in `get_output/artifact` if needed

Reply:
This means that the quantization recipe doesn't necessarily have configs to use with quantize_ but might still have pt2e quantization. In that case it doesn't make sense to throw an error right?

Reply:
Not sure I follow what you mean.